yarn.lock

```diff
@@ -6339,6 +6360,27 @@ __metadata:
   languageName: node
   linkType: hard
 
+"c8@npm:^9.1.0":
```

Comment:
Could you bump this to 10.x so that we can us the same version that's already used everywhere else?

Reply:
Done.